### PR TITLE
Add out-of-iframe app generator

### PR DESCRIPTION
**Pull request by naamagr**
### 🔦 Summary
This adds out-of-iframe tempalte to `create-yoshi-app`, select the `out-of-iframe` template

### 🗺️ Test Plan
* checkout branch
* run `npm run create-yoshi-app:dev`
* Create an app in dev center:
screen shot 2019-01-27 at 20 45 35
* Install it on some of your sites in editor and assert it works:
screen shot 2019-01-28 at 10 55 33
* And also it's setting panel works:
screen shot 2019-01-28 at 10 55 27

### CODE REVIEW

File: packages/create-yoshi-app/templates/out-of-iframe-typescript/src/platform/viewerScript.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { exampleWidgetControllerFactory } from './exampleWidgetController';
+import {
+  IWidgetController,
+  IWidgetControllerConfig,
+} from '@wix/native-components-infra/dist/src/types/types';
+
+function createControllers(
+  controllersConfig: IWidgetControllerConfig[],
+): Promise<IWidgetController>[] {
+  return [exampleWidgetControllerFactory(controllersConfig[0])];
```
Comment (yurynix) [inline packages/create-yoshi-app/templates/out-of-iframe-typescript/src/platform/viewerScript.ts]: Probably should be a `.map`:
https://github.com/wix-private/form-builder/blob/483781f1393920c0fc2cbfd04ab7c3b4c1b341a3/src/viewer-app/viewer-app-impl.ts#L320

File: packages/create-yoshi-app/templates/out-of-iframe-typescript/src/config/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import memoize from 'lodash/memoize';
+import i18next from 'i18next';
+
+export const i18nInstance = i18next.createInstance();
+
+export default memoize(function i18n(locale: string) {
```
Comment (yairhaimo) [inline packages/create-yoshi-app/templates/out-of-iframe-typescript/src/config/i18n.ts]: Why do we need to memoize this?

**Comment by ronami** [comment]
Can this be closed now that #1217 has been merged?

**Comment by ranyitz** [comment]
@ronami that's correct, Closing.

**Pull request closed without merging**